docs/user/pam-authentication.md

```
@@ -0,0 +1,161 @@
+
+For security reasons, the PAM issuer manages its own users within its container. This isolation ensures that authentication credentials are contained within the PAM issuer service and do not affect the host system.
+
+To add a user to the PAM issuer:
+
+1. **Create a new user:**
+
+```bash
+sudo podman exec flightctl-pam-issuer adduser <USER>
+```
+
+2. **Set the user's password:**
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)